### ISSUE TITLE
Openvpn 2.5.2 does not like "script-security" parameter

### ISSUE THREAD

**Issue by Matt812g** [haugene/docker-transmission-openvpn#2697]
### Is there a pinned issue for this?

- [X] I have read the pinned issues and could not find my issue

### Is there an existing or similar issue/discussion for this?

- [X] I have searched the existing issues
- [X] I have searched the existing discussions

### Is there any comment in the documentation for this?

- [X] I have read the documentation, especially the FAQ and Troubleshooting parts

### Is this related to a provider?

- [X] I have checked the provider repo for issues
- [X] My issue is NOT related to a provider

### Are you using the latest release?

- [X] I am using the latest release

### Have you tried using the dev branch latest?

- [X] I have tried using dev branch

### Docker run config used

```
docker run --rm -it --cap-add=NET_ADMIN \
  -e OPENVPN_PROVIDER=FREEVPN \
  -e OPENVPN_CONFIG=me-UDP-40000 \
  -e OPENVPN_USERNAME=freevpn.me \
  -e OPENVPN_PASSWORD=k2YbR6Ve2JBe \
  -e DISABLE_PORT_UPDATER=true \
  haugene/transmission-openvpn
```

### Current Behavior

Exits without running container when above command is run, with an error (see logs).

(Note that the creds are the free creds published here [https://freevpn.me/accounts/], so no security risk to put them in this issue.)

### Expected Behavior

It should run.

### How have you tried to solve the problem?

I researched why this is, so no need to try different things.  See "Anything Else" for research notes.

### Log output

> Found configs for FREEVPN in /config/vpn-configs-contrib/openvpn/freevpn, will replace current content in /etc/openvpn/freevpn
> Executing setup script for FREEVPN
> Starting OpenVPN using config me-UDP-40000.ovpn
> Modifying /etc/openvpn/freevpn/me-UDP-40000.ovpn for best behaviour in this container
> Modification: Point auth-user-pass option to the username/password file
> Modification: Change ca certificate path
> Modification: Change ping options
> Modification: Update/set resolv-retry to 15 seconds
> Modification: Change tls-crypt keyfile path
> Modification: Set output verbosity to 3
> Modification: Remap SIGUSR1 signal to SIGTERM, avoid OpenVPN restart loop
> Modification: Updating status for config failure detection
> Setting OpenVPN credentials...
> **Options error: Unrecognized option or missing or extra parameter(s) in [CMD-LINE]:1: script-security 2 --route-up /etc/openvpn/tunnelUp.sh --route-pre-down /etc/openvpn/tunnelDown.sh (2.5.5)**
> Use --help for more information.

### HW/SW Environment

```markdown
- OS: GUIX latest
- Docker: 2.5.2
```

### Anything else?

I anticipate that this will happen with any provider, as 'script-security' is even hardcoded into the non-provider scripts.  This openvpn option is needed in versions < 2.3, but is no longer accepted in versions >= 2.3.  See https://openvpn.net/community-resources/reference-manual-for-openvpn-2-4/, and search for 'script-security'.  It says:

> --script-security level
> This directive offers policy-level control over OpenVPN's usage of external programs and scripts. Lower level values are more restrictive, higher values are more permissive. Settings for level:0 -- Strictly no calling of external programs.
> 1 -- (Default) Only call built-in executables such as ifconfig, ip, route, or netsh.
> 2 -- Allow calling of built-in executables and user-defined scripts.
> 3 -- Allow passwords to be passed to scripts via environmental variables (potentially unsafe).  OpenVPN releases before v2.3 also supported a method flag which indicated how OpenVPN should call external commands and scripts.  This could be either execve or system. **As of OpenVPN 2.3, this flag is no longer accepted.**  In most *nix environments the execve() approach has been used without any issues. ...

If you have an old and not-up-to-date alpine, this may not affect you until you update it.  However, I just built mine, and alpine:latest comes with openvpn 2.5.2, which does not accept the 'script-security' option.

**Comment by pkishino** [comment]
check this please 
#2702

**Comment by Matt812g** [comment]
Huh.  I searched for script-security, not script_security.

Yep, seems same.  Closing as duplicate.

**Comment by Matt812g** [comment]
Oh, you mean check the PR.  Duh.

**Comment by Matt812g** [comment]
Looks good.  Thanks, @pkishino !

**Comment by pkishino** [comment]
Fixed in #2702
